retrofit/src/test/java/retrofit2/RetrofitTest.java

```diff
@@ -171,6 +171,37 @@
     assertSame(callFactory, two.callFactory());
   }
 
+  @Test public void builtInConvertersAbsentInCloneBuilder() {
+    Retrofit retrofit = new Retrofit.Builder()
+        .baseUrl(server.url("/"))
+        .build();
+
+    assertEquals(retrofit.converterFactories().size() - 1,
```

Comment:
should this just assert that it’s empty? Otherwise if we split the built in ones across two factories then this test is wrong

Reply:
That sounds like a much more robust test. b2b3b841